python-sdk/src/astro/sql/operators/load_file.py

```diff
@@ -77,7 +77,9 @@ def execute(self, context: Context) -> BaseTable | File:  # skipcq: PYL-W0613
         """
         if self.input_file.conn_id:
             check_if_connection_exists(self.input_file.conn_id)
-
+        if self.output_table:
+            context["ti"].xcom_push(key="output_table_conn_id", value=str(self.output_table.conn_id))
```

Comment:
We shall add a comments here to remove it in future

Reply:
+1